modules/exploits/windows/smb/smb_relay.rb

```diff
@@ -353,6 +354,9 @@ def smb_cmd_negotiate(c, buff)
     # Record the remote process ID
     smb[:process_id] = pkt['Payload']['SMB'].v['ProcessID']
 
+    flags2 = pkt['Payload']['SMB'].v['Flags2']
+    extended_security = (flags2 & 0x800 == 0x800 ? true : false)
```

Comment:
@bcook-r7 I'm not sure of get your comment, sorry :( What I'm trying to get is:

```
2.0.0-p451 :001 > (0x800 & 0x800 == 0x800 ? true : false)
 => true
2.0.0-p451 :002 > (0x801 & 0x800 == 0x800 ? true : false)
 => true
2.0.0-p451 :003 > (0xc01 & 0x800 == 0x800 ? true : false)
 => true
2.0.0-p451 :004 > (0x101 & 0x800 == 0x800 ? true : false)
 => false
```

Is something wrong with it :?